doc/contributing.md

```diff
@@ -273,18 +203,232 @@ This installs your project in *editable* mode, meaning that changes made to the
+
+* Each tutorial should focus on a particular set of tasks that a user might want to
+  accomplish: plotting grids, interpolation, configuring the frame, projections, etc.
+* The tutorial code should be as simple as possible. Avoid using advanced/complex Python
+  features or abbreviations.
+* Explain the options and features in as much detail as possible. The gallery has
+  concise examples while the tutorials are detailed and full of text.
+* SI units should be used in the example code for tutorial plots.
+
+Note that the `Figure.show()` function needs to be called for a plot to be inserted into
+the documentation.
+
+### Editing the API Documentation
+
+The API documentation is built from the docstrings in the Python `*.py` files under
+the `pygmt/src/` and `/pygmt/datasets/` folders. **All docstrings** should follow the
+[numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).
+All functions/classes/methods should have docstrings with a full description of all
+arguments and return values.
+
+While the maximum line length for code is automatically set by *Black*, docstrings
+must be formatted manually. To play nicely with Jupyter and IPython, **keep docstrings
+limited to 79 characters** per line.
+
+### Standards for Example Code
+
+When editing documentation, use the following standards to demonstrate the example code:
+
+1. Python arguments, such as import statements, Boolean expressions, and function
+   arguments should be wrapped as ``code`` by using \`\` on both sides of the code.
+   Examples: \`\`import pygmt\`\` results in ``import pygmt``, \`\`True\`\` results in `True`,
+    \`\`style="v"\`\` results in `style="v"`.
+2. Literal GMT arguments should be **bold** by wrapping the arguments with \*\*
+   (two asterisks) on both sides. The argument description should be in *italicized*
+   with \* (single asterisk) on both sides.
+   Examples: `**+l**\ *label*` results in **+l***label*, `**05m**` results in **05m**.
+3. Optional arguments are wrapped with [ ] (square brackets).
+4. Arguments that are mutually exclusive are separated with a | (bar) to denote "or".
+5. Default arguments for parameters and configuration settings are wrapped
+   with [ ] (square brackers) with the prefix "Default is". Example: [Default is
+   **p**].
+
+### Cross-referencing with Sphinx
+
+The API reference is manually assembled in `doc/api/index.rst`.
+The *autodoc* sphinx extension will automatically create pages for each
+function/class/module listed there.
+
+You can reference functions, classes, methods, and modules from anywhere
+(including docstrings) using:
+
+- <code>:func:\`package.module.function\`</code>
+- <code>:class:\`package.module.class\`</code>
+- <code>:meth:\`package.module.method\`</code>
+- <code>:mod:\`package.module\`</code>
+
+An example would be to use
+<code>:meth:\`pygmt.Figure.grdview\`</code> to link
+to https://www.pygmt.org/latest/api/generated/pygmt.Figure.grdview.html.
```

Comment:
The **four links** in this paragraph do not look like super links. I guess the reason is that we use the markdown language here:

![http](https://user-images.githubusercontent.com/50591376/123504991-1d967880-d68f-11eb-860d-921d00f81fca.png)

There are also **three dashub links** in **Testing Plots** section that have the same issue.

---

See [Citing PyGMT](https://www.pygmt.org/latest/index.html#citing-pygmt) on the landing page, which uses ReST:
![citing](https://user-images.githubusercontent.com/50591376/123505063-8a117780-d68f-11eb-94e3-56289521f262.png)

Reply:
I did not look through all the comments to double check, but I remember this being a intentional design choice in https://github.com/GenericMappingTools/pygmt/pull/802.

Reply:
I added hyperlinks while still showing the full url in https://github.com/GenericMappingTools/pygmt/pull/1339/commits/82e04ec681efc49e64a174f68ecd08f7d485d38e.